server/channels/api4/user.go

```diff
@@ -71,7 +71,7 @@ func (api *API) InitUser() {
 	api.BaseRoutes.UserByUsername.Handle("", api.APISessionRequired(getUserByUsername)).Methods("GET")
 	api.BaseRoutes.UserByEmail.Handle("", api.APISessionRequired(getUserByEmail)).Methods("GET")
 
-	api.BaseRoutes.User.Handle("/sessions", api.APISessionRequired(getSessions)).Methods("GET")
+	api.BaseRoutes.User.Handle("/sessions", api.RateLimitedHandler(api.APISessionRequired(getSessions), model.RateLimitSettings{PerSec: model.NewInt(2), MaxBurst: model.NewInt(10)})).Methods("GET")
```

Comment:
Do we still need the rate limit now after the latest changes?

Reply:
Probably not, removed it, thanks.